src/rsqsim_api/rsqsim_api/catalogue/catalogue.py

```diff
@@ -127,6 +134,22 @@ def patch_slip(self, data_list: np.ndarray):
         self.check_list(data_list, data_type="d")
         self._patch_slip = data_list
 
+    @property
+    def accumulated_slip(self):
+        return self._accumulated_slip
+
+    @accumulated_slip.setter
+    def accumulated_slip(self, data_list: dict):
+        self._accumulated_slip = data_list
+
+    @property
+    def event_mean_slip(self):
+        return self._event_mean_slip
```

Comment:
```suggestion
    @property
    def event_mean_slip(self):
        if self._event_mean_slip is None:
            self.assign_event_mean_slip()
        return self._event_mean_slip
```

Reply:
This won't work because assign_event_mean_slip needs fault model as input, unless there's a way of accessing event objects without a specific fault model?

Reply:
Good point @Camilla-P... Maybe leave things how they were